adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -454,21 +454,27 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] extends Logging {
    * @param environment A map containing environment variable/value pairs to set
    *   in the environment for the newly created process. Default is empty.
    * @param flankSize Number of bases to flank each command invocation by.
+   * @param optTimeout An optional parameter specifying how long to let a single
```

Comment:
Add doc that this is in seconds, or alternatively allow the caller to specify the `TimeUnit`